newton/examples/selection/example_selection_multiple.py

```
@@ -0,0 +1,302 @@
+
+    def render(self):
+        self.viewer.begin_frame(self.sim_time)
+        self.viewer.log_state(self.state_0)
+        self.viewer.end_frame()
+
+    def test_final(self):
+        newton.examples.test_body_state(
+            self.model,
+            self.state_0,
+            "all bodies are above the ground",
+            lambda q, qd: q[2] > 0.01,
```

Ruff (0.14.13)

Unused lambda argument: `qd`

(ARG005)